lib/taurus/qt/qtcore/util/emitter.py

```diff
@@ -196,7 +243,8 @@ def getQueue(self):
 
     def getDone(self):
         """ Returns % of done tasks in 0-1 range """
-        return self._done / (self._done + self.getQueue().qsize()) if self._done else 0.
+        return self._done / (self._done + self.getQueue().qsize()) \
```

Comment:
Please use a more readable if-else construct.
(PEP8 is not only about putting inserting line breaks at the middle of over-complex constructs) ;)